kolibri/core/assets/src/components_demo.js

```diff
@@ -109,6 +109,160 @@ var TextInputWithTagDisplay = Mn.LayoutView.extend({
     }
 });
 
+
+// This is a deeply nested view, used below.
+// Marionette requires deeply-nested views to be defined before they're used in their containing views.
```

Comment:
This is also idiomatic Javascript - jshint throws a fit if this is not the case also.